core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.list.listOfHashMaps;
```

Comment:
Hi @azhwani , we know that we could use camel case here but our convention is lower case. So could you rename the package to all lower case?